### PR TITLE
Fix EmojiStats on mobile

### PR DESCRIPTION
**Pull request by Scott-Caldwell**
Removes emoji URLs to avoid breaking the formatting on mobile. Switches the default command to the "guild" version, introduces an "all" version.

### CODE REVIEW

**Comment by calledude** [comment]
I wonder if we should provide a link for the emojis that aren't available to the bot in the `:emoji:` block that comes after it.
bild
so that the user still has some kind of reference as to what that emoji actually is

File: Modix.Bot/Modules/EmojiStatsModule.cs [hunk 1]
```diff
@@ -144,8 +144,8 @@ private async Task<EmbedBuilder> BuildEmojiStatEmbedAsync(SortDirection sortDire
                 var canAccess = ((SocketSelfUser)Context.Client.CurrentUser).CanAccessEmoji(emoji);
 
                 var emojiFormatted = canAccess
-                    ? Format.Url(emoji.ToString(), EmojiUtilities.GetUrl(emoji.ToString()))
-                    : Format.Url("❔", emoji.Url);
+                    ? emoji.ToString()
```
Comment (calledude) [inline Modix.Bot/Modules/EmojiStatsModule.cs]: Same as other comment

File: Modix.Bot/Modules/EmojiStatsModule.cs [hunk 2]
```diff
@@ -91,8 +91,8 @@ await ReplyAsync(embed: new EmbedBuilder()
             var guildStats = await _emojiRepository.GetGuildStatsAsync(guildId);
 
             var emojiFormatted = ((SocketSelfUser)Context.Client.CurrentUser).CanAccessEmoji(ephemeralEmoji)
-                    ? Format.Url(ephemeralEmoji.ToString(), EmojiUtilities.GetUrl(ephemeralEmoji.ToString()))
-                    : Format.Url("❔", ephemeralEmoji.Url);
+                    ? ephemeralEmoji.ToString()
```
Comment (calledude) [inline Modix.Bot/Modules/EmojiStatsModule.cs]: Just double-checking, is this ternary necessary?

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/EmojiStatsModule.cs]: How else would you do it?

Comment (calledude) [reply Modix.Bot/Modules/EmojiStatsModule.cs]: Assuming that `GetGuildStatsAsync()` does what I think it does, wouldn't the results be guild-specific, and as such we can ensure that the bot can see them all? Or is there a scenario where some guild-emotes can't be accessed by the bot?

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/EmojiStatsModule.cs]: This method is used by both the guild version and the all version

Comment (calledude) [reply Modix.Bot/Modules/EmojiStatsModule.cs]: OH right, just reading the function name made me think it was guild-specific emotes, my bad :)

**Comment by Scott-Caldwell** [comment]
>I wonder if we should provide a link for the emojis that aren't available to the bot in the :emoji: block that comes after it.

Added in latest commit.

File: Modix.Bot/Modules/EmojiStatsModule.cs [hunk 3]
```diff
@@ -159,7 +159,7 @@ private async Task<EmbedBuilder> BuildEmojiStatEmbedAsync(SortDirection sortDire
                     .Append($" ({"use".ToQuantity(emojiStat.Uses)})")
                     .Append($" ({percentUsage.ToString("0.0")}%),")
                     .Append($" {perDay.ToString("0.0/day")}")
-                    .Append(canAccess ? string.Empty : $" (`:{emoji.Name}:`)")
+                    .Append(canAccess ? string.Empty : $" ({Format.Url($":{emoji.Name.Replace("_", "\\_")}:", emoji.Url)})")
```
Comment (calledude) [inline Modix.Bot/Modules/EmojiStatsModule.cs]: `$@`? that way you wouldn't have to deal with replacing

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/EmojiStatsModule.cs]: Fixed

**Pull request merged**